miden-tx/src/host/account_delta.rs

```diff
@@ -97,12 +86,12 @@ impl AccountVaultDeltaHandler {
         Ok(HostResponse::None)
     }
 
-    // CONSUMERS
+    // CONVERSIONS
     // --------------------------------------------------------------------------------------------
 
-    /// Consumes the [AccountVaultDeltaHandler] and returns the [AccountVaultDelta] that represents the
+    /// Consumes the this delta tracker and returns the [AccountVaultDelta] that represents the
     /// changes to the account's vault.
-    pub fn finalize(self) -> AccountVaultDelta {
+    pub fn into_vault_delta(self) -> AccountVaultDelta {
```

Comment:
Here we have removed the word `finalize` to turn it into `into_vault_delta`. 

 Just to make sure should we take a look at: https://github.com/0xPolygonMiden/miden-vm/blob/c097af7ca53da2026cec10d9245a006b42186643/processor/src/host/advice/providers.rs#L467-L497

 There we have kept the word `finalize`. What is the difference / similarities and why are we changing the naming.

Reply:
We can make the naming more specific here. Though, even in the VM, I don't love the `finalize` name. We should probably refactor how we do things in Miden VM in the future.